core/src/main/java/org/elasticsearch/index/mapper/TypeFieldMapper.java

```diff
@@ -88,29 +92,12 @@ public MetadataFieldMapper getDefault(MappedFieldType fieldType, ParserContext c
     }
 
     static final class TypeFieldType extends StringFieldType {
-        private boolean fielddata;
```

Comment:
actually this is just dead code removal: this property was never read or set. There is no change wrt fielddata on the `_type` field.